app/views/DR/ReportScreen/ReportScreenQuestions.js

```diff
@@ -56,7 +55,7 @@ export default function ReportScreenQuestions({ navigation }) {
   const sendDataToApi = async () => {
     let merged = {};
     if (answers.usage === 'mySelf') {
-      let userInfo = await AsyncStorage.getItem('UserPersonalInfo');
+      let userInfo = await GetStoreData('UserPersonalInfo', false);
```

Comment:
you should use constants instead of a simple string when using the key of the storage.